### ISSUE TITLE
Updated analysis: molecular-subtype-pathology: update 7316-2557 subtyping to DMG, H3 K28M 

### ISSUE THREAD

**Issue by kgaonkar6** [AlexsLemonade/OpenPBTA-analysis#905]
#### What analysis module should be updated and why?
BS_YB99Q6TN (WGS) and BS_5965SFPZ (which is the matched RNAseq sample) belonging to `sample_id== "7316-2557"` are K28M as per pathology feedback and is also found K28M in mutect2 calls.

#### What changes need to be made? Please provide enough detail for another participant to make the update.
Add : 

```r
compiled_df <- compiled_df %>%
  mutate(
    integrated_diagnosis = case_when(
      sample_id == "7316-2557" ~ "Diffuse midline glioma, H3 K28-mutant",
      TRUE ~ integrated_diagnosis
    ),
    short_histology = case_when(
      sample_id == "7316-2557" ~ "HGAT",
      TRUE ~ short_histology
    ),
    broad_histology = case_when(
      sample_id == "7316-2557" ~ "Diffuse astrocytic and oligodendroglial tumor",
      TRUE ~ broad_histology
    ),
    molecular_subtype = case_when(
      sample_id == "7316-2557" ~ "DMG, H3 K28",
      TRUE ~ molecular_subtype
    ),
    Notes = case_when(
      sample_id == "7316-2557" ~ "Updated via OpenPBTA subtyping",
      TRUE ~ Notes
    )
  )
```

#### What input data should be used? Which data were used in the version being updated?
pbta-histologies-base.tsv v18

#### When do you expect the revised analysis will be completed?
TBD

#### Who will complete the updated analysis?
@kgaonkar6

**Comment by jharenza** [comment]
@kgaonkar6 I can take care of this issue this weekend.

**Comment by jharenza** [comment]
@kgaonkar6 see #923 - haha, I thought this would be quick and easy 😄

**Comment by jharenza** [comment]
closed with #935
